test/views/behaviour.js

```diff
@@ -0,0 +1,193 @@
+
+/**
+ * Behaviour of the Echidna testbed.
+ *
+ * @author Antonio <antonio@w3.org>
+ */
+
+'use strict';
```

Comment:
Makes complete sense, I'll try to remember this convention when editing code.

By the way... please remove line 1, empty :D Don't tell me this improves readability :-)